api/apps/articles/model/save.coffee

```diff
@@ -62,11 +62,13 @@ getDescription = (article) =>
   text
 
 removeStopWords = (title) ->
-  title = title.replace(/[.,\/#!$%\^&\*;:{}=\—_`’~()]/g," ")
-  newTitle = _.difference(title.split(' '), stopWords.stopWords).join(' ')
-  return newTitle
+  title = title.replace(/[0-9.\/#!$%\^\*;{}=_`’~()]/g,"")
```

Comment:
I see, thanks! If it doesn't affect drafts that works for me but in general I don't know if numbers should be a stop word. @owendodd can you comment?